GameDefinition/PLAYING.md

```diff
@@ -527,6 +539,99 @@ influence the speed of HP attrition and if it outweighs the benefits
 of exploration.
 
 
+Managing inventory and crafting
+-------------------------------
+
+All items belonging to your team are stored either in the character's local
+inventory (called the character's outfit) or in the shared inventory that
+is accessible by every team member (called the team stash). Items that are
+in the character's outfit can be accessed by or are beneficial for this
+particular character only. Stashed items can be accessed by the team members
+(unless the character is injured or panicking). But no one has a direct
+advantage from having the items in the shared inventory.
+The stashed items are exactly stashed, they are not simply laying on
+the ground (although the stash has a physical location on the map). You
+cannot operate the items from the stash, as if these items are just on
+the ground directly. To collect an item from the stash or add it there,
+you should enter the stash management menu by pressing the 'I'.
+
+Obviously, if the item is beneficial (a piece of armor, weapon, or a bonus
+item), it is reasonable to put it into a particular character's outfit.
+However, you should remember that the outfit is limited to any 10 (ten)
+items only. On the contrary, if the item is not required directly (ranged
+projectiles, medkits, ampoules with drugs, or food), it may be stored in
+the shared stash.
+The shared stash is of unlimited size. It has a physical location on
+the map and may be looted by your enemies. So, it is wise to keep it safe
+or change its location depending on the current situation. It would be
+a pity to lose your belongings, valuables, and powerful projectiles by not
+taking care of them. Enemy factions may have their stashes on the map,
+and you may loot their stashes too. Enemy units may have items in their
+outfits, and sometimes they may have some items stashed on the ground.
+To relocate the stash, first, you need to remove ALL of the items from it
+anywhere you want to make it empty. Then, any stashed item (or a pile
+of items) will automatically become the new stash. The process is a bit
+long, so you should plan stash relocation in advance.
```

Comment:
I don't remember, but doesn't it take 2 turns? That's risky if standing next to an enemy, but not "long".

Reply:
1. Remove all from the stash.
2. Stash all.
That's almost that place from where I contacted you on the game website :)

Reply:
So is it just 2 turns? You can check the turn counter at the top of history display before and after.

Reply:
Well, the long crawl (with a HUGE stash with over 3 pages of items) is over, and relocation of a small stash takes 2 turns.

Reply:
Thank you for the confirmation. Otherwise, it'd probably require a change.

So it's rather "a couple of turns" than "several turns", right? Pardon my pedantry: it's an occupational disease.

Reply:
No problem. 2 turns so far.